CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+
+## [1.51](https://github.com/brave/brave-ios/releases/tag/v1.51)
+
+### Web3
+
+- Added network icons to be shown for native assets. ([#7123](https://github.com/brave/brave-ios/issues/7123))
+- Implemented Market Tab in Brave Wallet. ([#7202](https://github.com/brave/brave-ios/issues/7202))
```

Comment:
nit: don't think `Tab` needs to be capitalized as the `tab` one below isn't either. Should be consistent 👍